ibm/mas_devops/playbooks/oneclick_add_mso.yml

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
No objection to adding this, but if we are delivering the playbook the doc for the playbook must also be created explaining where this fits into the bigger picture (e.g. what other oneclick playbooks must be ran first)